app/views/forecasts/show.html.erb

```diff
@@ -5,4 +5,3 @@
 <%= render partial: "sharing" %>
 <%= render partial: "learning" %>
 <%= render partial: "subscribe" %>
-<%= javascript_include_tag "application", "data-turbo-track": "reload", type: "module" %>
```

Comment:
I moved it in here because when it was in the layout file, our application.js handlers were running on the index page, which obviously didn't have any of the elements it was looking for. I added some guards in that file but it felt a bit janky. So I moved the tag in here. But now that we do genuinely have multiple pages, maybe the guard approach is best for now?

Reply:
I think ultimately we want all of the application.js code to end up in Stimulus controllers. There's not much left in there now.